modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectEntryLocalServiceImpl.java

```diff
@@ -1736,7 +1738,10 @@ private GroupByStep _getOneToManyObjectEntriesGroupByStep(
 				}
 			).and(
 				() -> {
-					if (PermissionThreadLocal.getPermissionChecker() == null) {
+					if (bypassPermission ||
```

Comment:
I didn't find this modification related to M-M relationship...

Reply:
I missed this detail. Thanks for noticing it!